src/main/paradox/docs/additional-info/architecture/components.md

```diff
@@ -1,6 +1,6 @@
 # Components
 
-![Component View](components.png)
+![Component View](components1.png )
```

Comment:
```suggestion
![Component View](components1.png)
```

Reply:
Also I don't see the new file `components1.png`

Reply:
is in the pull request #614